### DISCUSSION TITLE
Can't verify myself as a student

### DISCUSSION THREAD

**Discussion by spadefuljacknoir** [community/community#50938] [GitHub Education]
Hello. I have my physical ID from my university and i tried to show it to verify myself as a student (it is on my native language, Ukrainian).
I also have a university mail, but Campus always rejects my efforts, saying that, maybe, you need to fix your bio or the name and surname in your account. What can I do?

![изображение](https://user-images.githubusercontent.com/114944216/227579095-cb686583-dd5c-4f39-94da-1330368f88af.png)

**Comment by samus-aran** [comment]
Hi @spadefuljacknoir 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
